src/lib/import-export/import/importers/importer.ts

```diff
@@ -310,3 +317,99 @@ export class SQLImporter extends BaseImporter {
 		}
 	}
 }
+
+export class WpressImporter extends BaseBackupImporter {
+	protected async parseMetaFile(): Promise< MetaFileData > {
+		const packageJsonPath = path.join( this.backup.extractionDirectory, 'package.json' );
+		try {
+			const packageContent = await fsPromises.readFile( packageJsonPath, 'utf8' );
+			const {
+				Template: template = '',
+				Stylesheet: stylesheet = '',
+				Plugins: plugins = [],
+			} = JSON.parse( packageContent );
+			return { template, stylesheet, plugins };
+		} catch ( error ) {
+			console.error( 'Error reading package.json:', error );
+			return { template: '', stylesheet: '', plugins: [] };
+		}
+	}
+
+	protected async prepareSqlFile( tmpPath: string ): Promise< void > {
+		const tempOutputPath = `${ tmpPath }.tmp`;
+		const readStream = createReadStream( tmpPath, 'utf8' );
+		const writeStream = createWriteStream( tempOutputPath, 'utf8' );
+
+		const rl = createInterface( {
+			input: readStream,
+			crlfDelay: Infinity,
+		} );
+
+		rl.on( 'line', ( line: string ) => {
+			writeStream.write( line.replace( /SERVMASK_PREFIX/g, 'wp' ) + '\n' );
+		} );
+
+		await new Promise( ( resolve, reject ) => {
+			rl.on( 'close', resolve );
+			rl.on( 'error', reject );
+		} );
+
+		await new Promise( ( resolve, reject ) => {
+			writeStream.end( resolve );
+			writeStream.on( 'error', reject );
+		} );
+
+		await fsPromises.rename( tempOutputPath, tmpPath );
+	}
+
+	protected async addSqlToSetTheme( sqlFiles: string[] ): Promise< void > {
+		const { template, stylesheet } = this.meta || {};
+		if ( ! template || ! stylesheet ) {
+			return;
+		}
+
+		const themeUpdateSql = `
+				UPDATE wp_options SET option_value = '${ template }' WHERE option_name = 'template';
+				UPDATE wp_options SET option_value = '${ stylesheet }' WHERE option_name = 'stylesheet';
+			`;
+		const sqliteSetThemePath = path.join(
+			this.backup.extractionDirectory,
+			'studio-wpress-theme.sql'
+		);
+		await fsPromises.writeFile( sqliteSetThemePath, themeUpdateSql );
+		sqlFiles.push( sqliteSetThemePath );
+	}
+
+	protected async addSqlToActivatePlugins( sqlFiles: string[] ): Promise< void > {
+		const { plugins = [] } = this.meta || {};
+		if ( plugins.length === 0 ) {
+			return;
+		}
+
+		const serializedPlugins = serializePlugins( plugins );
+		const activatePluginsSql = `
+			INSERT INTO wp_options (option_name, option_value, autoload) VALUES ('active_plugins', '${ serializedPlugins }', 'yes');
```

Comment:
Is there any benefit from using SQL queries over WP CLI calls?

Reply:
I initially thought about using wp-cli, but it turns out the site isn't ready until it's started for the first time. That's why I opted to run SQL queries, which is one of the steps in the import process.

Reply:
@wojtekn , I've pushed a fix to make it work in your case.
The error is that your SQL had already an empty options for `active_plugins`. It's fixed on https://github.com/Automattic/studio/pull/497/commits/ce7cbd34b210587ea7159818a561e77dc0befb6a